### PR TITLE
Switch from swiftformat to swift-format

### PR DESCRIPTION
**Pull request by armandgrillet**
This PR is made to close https://github.com/swift-server/swift-prometheus/issues/94.

The code has been tested using `swift-format lint --configuration .swift-format --recursive Sources`.

`Tests/PrometheusTests` should be formatted as well but contains too many things to change for now, this is why I have started with only `Sources` and added `"lineLength": 1000` to limit the number of changes to do.

### CODE REVIEW

File: scripts/soundness.sh [hunk 1]
```diff
@@ -37,17 +37,14 @@ if git grep --color=never -i "${unacceptable_terms[@]}" > /dev/null; then
...
-#SECOND_OUT="$(git status --porcelain)"
-#if [[ "$FIRST_OUT" != "$SECOND_OUT" ]]; then
-#  printf "\033[0;31mformatting issues!\033[0m\n"
-#  git --no-pager diff
-#  exit 1
-#else
-#  printf "\033[0;32mokay.\033[0m\n"
-#fi
+printf "=> Checking format... "
+output=$(swift-format lint --configuration .swift-format --recursive Sources 2>&1)  # Capture the command's output
```
Comment (ktoso) [inline scripts/soundness.sh:41]: Tests/ directory too

**Comment by ktoso** [comment]
I think it's okey to take the hit and do it all at once -- format the tests too; it'd be weird to only have a part of the codebase use a formatter

**Comment by ktoso** [comment]
@swift-server-bot add to allowlist

File: scripts/soundness.sh (hunk 1 above)
Comment (armandgrillet) [reply scripts/soundness.sh]: Done ✅

**Comment by armandgrillet** [comment]
Not sure why CI is failing, locally:

```
➜ swift-prometheus (armandgrillet/swift-format) ✔ bash scripts/soundness.sh
=> Checking for unacceptable language... okay.
=> Checking format... okay.
=> Checking license headers
   * swift-or-c... okay.
   * bash... okay.
   * dtrace... okay.
=> Checking for imports of umbrella NIO module... okay.
```

On https://ci.swiftserver.group/job/swift-prometheus-soundness-prb/25/console :
```
+ ./scripts/soundness.sh
=> Checking for unacceptable language... [0;32mokay.[0m
=> Checking format... 127
Build step 'Execute shell' marked build as failure
```

Does someone have access to more logs?

File: .swift-format
Comment (FranzBusch) [inline .swift-format:1]: Can you use the config file from `swift-certificates` please: https://github.com/apple/swift-certificates/blob/main/.swift-format

**Review by FranzBusch — APPROVED** [review]

**Comment by armandgrillet** [comment]
All checks now pass apart from `pull request validation (soundness)`. Running it locally works though:
```
➜ swift-prometheus (armandgrillet/swift-format) ✔ bash scripts/soundness.sh
=> Checking for unacceptable language... okay.
=> Checking format... okay.
=> Checking license headers
   * swift-or-c... okay.
   * bash... okay.
   * dtrace... okay.
=> Checking for imports of umbrella NIO module... okay.
```
Could someone with better access rights tell me what the CI step is running so that I can try to reproduce locally? Right now the output on Jenkins does not give me the details I need to pass this last check.

**Comment by ktoso** [comment]
The way to reproduce CI is:

- docker-compose -f docker/docker-compose.yaml -p swift-prometheus-soundness-prb build --pull
- docker-compose -f docker/docker-compose.yaml -p swift-prometheus-soundness-prb run --rm soundness

hope this helps?

**Review by FranzBusch — APPROVED** [review]

**Pull request merged**